imports/lib/dingwordLogic.ts

```diff
@@ -26,16 +26,18 @@ export function normalizedMessageDingsUserByDingword(
 }
 
 export function messageDingsUser(chatMessage: PartialChatMessageType, user: Meteor.User): boolean {
+  if (chatMessage.sender === user._id || chatMessage.sender === undefined) {
```

Comment:
This function is currently only used client-side.  I wound up with slightly separate functions here because the serverside code benefits from not re-normalizing the message for every user that wants to check if their dingwords are triggered, so it uses the two functions above directly.